fastlane/Fastfile

```diff
@@ -155,6 +155,34 @@ platform :ios do
     emerge(emerge_args)
   end
 
+  desc "Build Maestro App"
+  lane :build_maestro_app do |options|
+    # Find a simulator (first available iPhone/iPad)
+    device_id = sh("xcrun simctl list devices available | grep -E 'iPhone' | grep -Eo '[A-F0-9\\-]{36}' | tail -n1").strip
+    UI.user_error!("❌ No available simulator found!") if device_id.empty?
+
+    # Boot it (safe if already booted)
+    sh("xcrun simctl boot #{device_id}") rescue nil
```

Comment:
Would it be better to start the simulators from CircleCI? They have jobs that do this and other checks

Reply:
Yeah saw that after implementing this. I'll try it out